.github/workflows/build-aws-hub.yaml

```diff
@@ -0,0 +1,75 @@
+name: Deploy AWS Hub
+on:
+  push:
+    branches:
+      - staging
+      #- prod
+    paths:
+      - 'deployments/icesat2/**'
+      - 'pangeo-deploy/**'
+    paths_ignore:
+      - 'deployments/icesat2/image/binder/*'
+
+env:
+  HUBPLOY_IMAGE: docker://yuvipanda/hubploy:20200826083951674280
+
+jobs:
+  deploy_aws:
+    name: Deploy AWS Hub
+    # This job runs on Linux
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: $HUBPLOY_IMAGE
```

Comment:
i wonder what it would take to package hubploy as an action to use the standard pattern like `-uses: hubploy/hubploy@v1` (https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/creating-a-docker-container-action).

Reply:
@yuvipanda , do you have interest in this? I'm wondering if that would make it easier to use / manage `hubploy` for people. Currently, some pieces of the action feel a bit hacky, see things like from [IP whitelisting](https://github.com/pangeo-data/pangeo-cloud-federation/pull/754/files#diff-193e3aaa151c393bc9929d2de82aac8fR51-R58):

```
with:
  entrypoint: /bin/bash
  args: >
    -c "code &&
    more code &&
    end of code"
```